### ISSUE TITLE
Write Flow section in Quickstart needs more details about location of sample data

### ISSUE THREAD

**Issue by rohithdv** [fybrik/fybrik#1705]
In the [Write Flow usecase in Quick start](https://fybrik.io/v1.0/samples/notebook-write/) , we need to clearly specify how to copy the data needed by the notebook pod for doing the write. Currently it is mentioned to download and extract the file (for eg: [PS_20174392719_1491204439457_log.csv](https://raw.githubusercontent.com/fybrik/fybrik/master/samples/notebook/PS_20174392719_1491204439457_log.csv))  to your machine. But this is not enough. The file needs to be copied to the pod from the local machine for the notebook pod to be able to read. So we need to mention how to copy the file to the running notebook pod in the quickstart (maybe using [kubectl cp](https://www.cloudytuts.com/tutorials/kubernetes/how-to-copy-files-to-a-pod-container-in-kubernetes/) command).

**Comment by roytman** [comment]
@rohithdv , we don't require copying the file to the notebook pod. It should be on the same computer where a user runs the browser with the notebook (his laptop).
The notebook via browser accesses and uploads the file.
 
Do you think we need more explanations ?

**Comment by shlomitk1** [comment]
@roytman I think more explanations are required. Let's say I've downloaded the file and saved it in /tmp. What should I write instead of `file_path = "/path/to/PS_20174392719_1491204439457_log.csv"` s.t. the file on the host machine will be accessed and not the file in the container file system?

**Comment by rohithdv** [comment]
I tried the following.

1. Connect from my local machine to my VM instance. 
2. Run the  quickstart related pods related to write flow on the VM on kind cluster 
3. Open` localhost:8080 `on my local machine browser (which in turn port forwards to 8080 port on my VM)
4. If I provide a path on my VM to the csv file in the notebook, the pod was unable to access it. I was getting the following error: 
<img width="926" alt="image (5)" src="https://user-images.githubusercontent.com/59818576/186613595-69d2d6e7-ca9e-4925-b575-46810b73ba63.png">
<img width="954" alt="image (4)" src="https://user-images.githubusercontent.com/59818576/186613621-4f4df5fe-a368-4d5e-8be4-bbebecbfdda7.png">

6. Then I did `kubectl cp /data/rohith/v0.6.2/fybrik/rohith-fork/fybrik/samples/notebook fybrik-notebook-sample/my-notebook-write-848c968647-j8pn4:/tmp`. Then it was able to access the csv file when I provided `/tmp/notebook/PS_20174392719_1491204439457_log.csv` as the file location in the notebook.  
7.   However if I changed to some other location (`/sampledata` instead of` /tmp`) also, I was getting an error as given below.  
<img width="1344" alt="image (6)" src="https://user-images.githubusercontent.com/59818576/186613730-4634ac41-5322-4ade-bf70-86f2ab682884.png">

In my view, it is maybe better to clarify how to specify the location of the csv in more detail in the documentation. Thanks.
